### PR TITLE
feat: history-video-added

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Features include in this PR:
- On clicking, video cards are going to history page
- You can delete any particular video card from history page.
- On clicking CLEAR_ALL button, the whole history page will get cleared.

### CODE REVIEW

File: src/pages/History/history.jsx [hunk 1]
```diff
@@ -35,22 +43,23 @@ return (
...
                     null
                     }
-                    <p className="card-content">By Megha Joshi</p>
+                    <p className="card-content">{historyVideo.creator}</p>
                     <div className="card-footer">
-                        <p className="card-content">3K views</p>
-                        <p className="card-content">3 June, 2022</p>
+                        <p className="card-content">{historyVideo.views}</p>
+                        <p className="card-content">{historyVideo.date}</p>
+                        {/* <span onClick={() => removeVideoFromHistory(historyVideo._id, token)}><i class="fad fa-trash-alt"></i></span> */}
```
Comment (zoyatanweer) [inline src/pages/History/history.jsx:58]: remove if not required

**Review by zoyatanweer — APPROVED** [review]
- code looks great
- file structure is good
- good to merge

File: src/pages/History/history.jsx (hunk 1 above)
Comment (Megha-Joshi) [reply src/pages/History/history.jsx:58]: Okay

**Review by Jia-A — APPROVED** [review]
Everything looks good to me.
Code is structured and naming conventions make code understandable.
You can merge it.

**Pull request merged**